aws-javascript/package.json

```diff
@@ -3,7 +3,7 @@
     "main": "index.js",
     "dependencies": {
         "@pulumi/pulumi": "^3.0.0",
-        "@pulumi/aws": "^5.0.0",
-        "@pulumi/awsx": "^1.0.0"
+        "@pulumi/aws": "^6.0.0",
+        "@pulumi/awsx": "2.0.0-beta.3"
```

Comment:
Is it possible to run acceptance tests on the PR and then merge only after awsx has come out?

Reply:
Yes, that's the plan.